src/main/resources/db/migration/all/20231221142033__backfill_placement_application_decision_made.sql

```diff
@@ -0,0 +1 @@
+UPDATE placement_applications pa SET decision_made_at = (SELECT min(pr.created_at) FROM placement_requests pr WHERE pr.placement_application_id = pa.id)
```

Comment:
Thinking we should only set this on placment_applications with a decision i.e:

```
UPDATE placement_applications pa SET decision_made_at = (SELECT min(pr.created_at) FROM placement_requests pr WHERE pr.placement_application_id = pa.id) WHERE decision IS NOT NULL
```

Reply:
oh yeah, good call. fixed